documentation/running_bolt_commands.md

```diff
@@ -1,418 +1,640 @@
-    command in single quotation marks:
-    ```shell script
-    bolt command run 'echo $HOME' --targets web5.mydomain.edu,web6.mydomain.edu
-    ```
-    ```shell script
-    bolt command run "netstat -an | grep 'tcp.*LISTEN'" --targets web5.mydomain.edu,web6.mydomain.edu
-    ```
+```shell
+bolt help
+```
 
--   To run a cross-platform command:
-    ```shell script
-    bolt command run "echo 'hello world'"
-    ```
+📖 **Related information**
 
--   To read a command from a file, pass the command as an `@` symbol followed by
-    the path to the file:
+- [Bolt command reference](bolt_command_reference.md)
 
-    ```
-    bolt command run @command.txt --targets servers
-    ```
+### PowerShell cmdlets
 
-    For Windows PowerShell, add single quotation marks to define the file:
+If you are running Bolt from a Windows controller, you can use PowerShell
+cmdlets to run Bolt commands. The PowerShell cmdlets are defined in a
+PowerShell module that ships with Bolt. Each cmdlet follows verb-noun
+conventions.
 
-    ```
-    bolt command run '@command.txt' --targets servers
-    ```
+For example, to `invoke` a `command` on a target, you can use the
+`Invoke-BoltCommand` cmdlet. You would then specify the command to invoke with
+the `-Command` named argument:
 
--   To read a command from `stdin`, pass the command as a single dash `-` and
-    pipe the command to Bolt:
+```powershell
+Invoke-BoltCommand -Command 'pwd' -Targets servers
+```
 
-    ```
-    <COMMAND> | bolt command run - --targets servers
-    ```
+To view a full list of available PowerShell cmdlets, use the `Get-Command`
+cmdlet:
 
-    For example, if you have a command in a text file:
+```powershell
+Get-Command -Module PuppetBolt
+```
 
-    ```
-    cat command.txt | bolt command run - --targets servers
-    ```
+### Specifying targets
 
-> **Note:** When connecting to Bolt over WinRM, the target must be 
-> configured to use SSL over port 5986. If the target is not configured
-> to use SSL, use the `--no-ssl` option to connect to the default WinRM 
-> port 5985.
+Many of Bolt's commands are used to connect to remote targets and execute
+actions on them. There are several ways that you can specify targets for a
+command, from listing them on the command line to reusing targets from the
+previous Bolt command.
 
-## Running commands with redirection or pipes
+#### Specifying targets
 
-When you run one-line commands that include redirection or pipes, pass `bash` or
-another shell as the command.
+The most common way to specify targets on the command line is with the
+`targets` option. This option accepts a comma-separated list of targets.
 
-Using a shell ensures that the one-liner is run as a single command and that it
-works correctly with `run-as`. For example, instead of `bolt command run "echo
-foo > /root/foo" --run-as root`, use `bolt command run "bash -c 'echo foo >
-/root/foo'" --run-as root`.
+- **Unix shells**
```

Comment:
Let's go with the style/formatting we used on the new modules page:

*\*nix shell command*
```shell
bolt project migrate
```

*Powershell cmdlet*
```shell
Update-BoltProject
```

I ended up using italics because bold and H4 look almost identical